modules/imgcodecs/test/test_read_write.cpp

```diff
@@ -194,7 +194,7 @@ const string exts[] = {
 #ifdef HAVE_JPEG
     "jpg",
 #endif
-#if defined(HAVE_JASPER) && defined(OPENCV_IMGCODECS_ENABLE_JASPER_TESTS)
+#if defined(OPENCV_IMGCODECS_ENABLE_JPEG2K_TESTS)
```

Comment:
Lets revert this condition until we will have encoder implementation.

Reply:
started an encoder branch on top of this one and moved it there
https://github.com/StefanBruens/opencv/tree/jpeg2000_openjpeg_encoder